### DISCUSSION TITLE
Moving windows to external display – when connected?

### DISCUSSION THREAD

**Discussion by stoltenhoff** [rxhanson/RectanglePro-Community#361] [Q&A]
I've done some research before and failed with other tools like Mosaic, too: Is there a way to put all open apps on a dedicated display depending from a scenario? In the office i'm working at my MacBook and an external display, at home i'm working at the MacBook alone. So i'm using for example Sketch on the large display – when available. Right now i'm moving every single window from display to display plus have to toggle around the full screen mode …
I experimented with the triggers and applications layouts but nothing is happening?  Is there a misconception in my idea?

**Comment by rxhanson** [comment]
Thanks for letting me know. There could be a timing issue with the application layout display connected trigger. If you manually perform a keyboard shortcut for your app layout after the display is connected, does the app layout get applied?
